### PR TITLE
add spec text for a String.prototype.isWellFormed

### PR DESCRIPTION
**Pull request by michaelficarra**

### CODE REVIEW

File: spec.html [hunk 1]
```diff
@@ -9,15 +10,16 @@
...
-  <p>This is an algorithm:</p>
-  <emu-alg>
-    1. Let _proposal_ be *undefined*.
-    1. If _proposal_ is ~accepted~, then
-      1. Let _stage_ be 0.
-    1. Else,
-      1. Let _stage_ be -1.
-    1. Return ? ToString(_proposal_).
-  </emu-alg>
+<emu-clause number="22.1.3" id="sec-properties-of-the-string-prototype-object">
```
Comment (ljharb) [inline spec.html:13]: Does this work? I'd expect "number" to only govern within the section, meaning you'd need this:

```suggestion
<emu-clause number="22">
  <h1>Text Processing</h1>

  <emu-clause number="1">
    <h1>String Objects</h1>

    <emu-clause number="3" id="sec-properties-of-the-string-prototype-object">
```

File: spec.html [hunk 2]
```diff
@@ -9,15 +10,16 @@
...
+  <h1>Properties of the String Prototype Object</h1>
+
+  <emu-clause id="sec-string.prototype.iswellformed">
```
Comment (ljharb) [inline spec.html:16]: and then
```suggestion
  <emu-clause id="sec-string.prototype.iswellformed" number="10">
```

**Review by ljharb — COMMENTED** [review]
Should the IsStringWellFormedUnicode AO have "static semantics:" removed?

File: spec.html (hunk 1 above)
Comment (michaelficarra) [reply spec.html:13]: It does work. See https://tc39.es/ecmarkup/#emu-clause

**Comment by michaelficarra** [comment]
@ljharb Sure when we integrate at stage 4, that's probably something we would consider.

File: spec.html (hunk 1 above)
Comment (ljharb) [reply spec.html:13]: very interesting, thanks

File: spec.html (hunk 2 above)
Comment (ljharb) [reply spec.html:16]: In that case, this should still have number 10, otherwise it'll be incorrectly marked as 22.1.3.1

Comment (michaelficarra) [reply spec.html:16]: Sure, we can do that.

**Review by ljharb — APPROVED** [review]
LGTM

**Pull request merged**